src/dos/dos_locale.h

```diff
@@ -415,6 +415,34 @@ struct CountryInfoEntry {
 	std::string GetMsgName() const;
 };
 
+// List of all the supported alphabets
+enum class Script : uint8_t {
+	Latin,
+	Arabic,
+	Armenian,
+	Cherokee,
```

Comment:
All Cherokee DOOM gamers will rejoice 😆

Reply:
I would prefer Futhark and Glagolitic script support, but there are no relevant CPI files and keyboard layouts in the FreeDOS packages :(

![image](https://github.com/user-attachments/assets/753dcac0-c7f9-4a73-809f-94641e1faa15)

Reply:
Well, thank goodness there isn't! 😆